src/keri/app/signaling.py

```diff
@@ -146,7 +146,7 @@ def expireDo(self, tymth=None, tock=0.0):
         _ = (yield self.tock)
 
         while True:  # loop checking for expired messages
-            now = datetime.datetime.now()
+            now = helping.nowUTC()
```

Comment:
datetime.datetime.now(): Returns the current local date and time.
helping.nowUTC(): Returns the current date and time in UTC.

maybe this needs to be `helping.nowIso8601()` instead?

Reply:
I tried `helping.nowIso8601()` first but `test_signaling.py` throws the error `TypeError: unsupported operand type(s) for -: 'str' and 'datetime.datetime'` on line 152 of `signaling.py`.
We need a datetime object as returned by `helping.nowUTC()`:
`Returns timezone aware datetime of current UTC time Convenience function that allows monkeypatching in tests to mock time`

Reply:
ahhh OK  - got it and this pairs with the `dt = dt if dt is not None else helping.nowIso8601()` update as they both now use UTC.